structuredheadings/plugin.js

```diff
@@ -1,5 +1,42 @@
 (function () {
 /*
+ * Style Config
+ */
+
+  CKEDITOR.config.autonumberBaseClass = "autonumber";
+  CKEDITOR.config.autonumberRestartClass = "autonumber-restart";
+  CKEDITOR.config.autonumberStyles = {
+    Narara: {
+      h1: "autonumber-N",
+      h2: "autonumber-a",
+      h3: "autonumber-r",
+      h4: "autonumber-a",
+      h5: "autonumber-r",
+      h6: "autonumber-a"
+    },
+    Aarara: {
+      h1: "autonumber-A",
+      h2: "autonumber-a",
+      h3: "autonumber-r",
+      h4: "autonumber-a",
+      h5: "autonumber-r",
+      h6: "autonumber-a"
+    },
+    RANaNa: {
+      h1: "autonumber-R",
+      h2: "autonumber-A",
+      h3: "autonumber-N",
+      h4: "autonumber-a",
+      h5: "autonumber-N",
+      h6: "autonumber-a"
+    }
+  };
```

Comment:
Yeah I realized that when I was typing my previous comment... I'll be adding that.